cura/Settings/MachineManager.py

```diff
@@ -847,6 +847,24 @@ def variantBuildplateUsable(self) -> bool:
 
         return result
 
+    @pyqtProperty(bool, notify = currentConfigurationChanged)
+    def variantCoreUsableForFactor4(self) -> bool:
+        """The selected core is usable if it is in second extruder of Factor4
+        """
+        result = True
+        if not self._global_container_stack:
+            return result
+        if not self.activeMachine.id.startswith("UltiMaker Factor 4"):
```

Comment:
this must really be this
```suggestion
        if self.activeMachine.definition.id != "ultimaker_factor4":
```
we want to check for equality against the definition id, this is constant no matter what machine instance you have